doc/classes/Engine.xml

```diff
@@ -163,6 +165,7 @@
 		<method name="get_singleton_list" qualifiers="const">
 			<return type="PackedStringArray" />
 			<description>
+				Returns a list of available singletons.
```

Comment:
Would be interesting to note whether this only includes built-in or also user-defined singletons.

Reply:
Yeah, it includes both, so no point in mentioning it. I only clarified user singletons where relevant.